components/customs/header.tsx

```diff
@@ -105,6 +107,7 @@ export const Header = () => {
               </ul>
             </NavigationMenuContent>
           </NavigationMenuItem>
+          {/*
```

Comment:
Commented out "About" and "Documentation". Didn't feature flag off the "about" page because it felt unecessary but please let me know if that's something we'd like and I'll do it.